### ISSUE TITLE
Feature request: Improve accessabilty in the keyboard itself, not only keyboard settings

### ISSUE THREAD

**Issue by Zvonimir-FUTO** [futo-org/android-keyboard#1515]
### Use Cases

> First, text-to-speech screenreader (called Talkback) doesn't work with your keyboard outside of practicing on it in the app.

### Proposal

Describe each element so Talkback can read it to the user.

### References

- Email: #11524

**Comment by abb128** [comment]
Would need more specific information... I always test with Talkback on every change and it seems to work fine. Each element gets described correctly

**Comment by abb128** [comment]
Going to close this as stale due to lack of response from user
